MonoGame.Framework/Platform/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -187,6 +187,30 @@ protected virtual void CreateDeviceResources()
                 creationFlags |= SharpDX.Direct3D11.DeviceCreationFlags.Debug;
             }
 
+#if WINDOWS_UAP
+            // Check if tearing is supported
+            if (PresentationParameters.PresentationInterval == PresentInterval.Immediate)
```

Comment:
This check should happen in `OnPresentationChanged` because CreateDeviceResources is only called when the GD is created (I think the rest of the code here is not 100% consistent either though).

Reply:
I moved it as you requested, it's still working on my sample as before.